src/vendor/wordpress/_wordpress.scss

```diff
@@ -0,0 +1,27 @@
+@use "../../mixins/button";
+@use "../../design-tokens/sizes.yml";
+
+/**
+ * Gutenberg block: Button
+ * Applies our pattern library styles to buttons generated via Gutenberg blocks.
+ */
+
+.wp-block-buttons {
+  display: flex;
+  flex-wrap: wrap;
+
+  & * {
+    margin: sizes.$edge_large / -4;
+    padding: sizes.$edge_large;
+  }
+}
```

Comment:
The gap seems very, very small and there is some extra space around the outer edge.

Here's what I would do: https://codepen.io/tylersticka/pen/f51fd10bbaa45abc9238ee833392db71?editors=1100